README.md

```diff
@@ -5,3 +5,39 @@ A file can fall into one of three categories.
 1. Files that are core to Salt and will remain in the Salt codebase
 2. Files that will be moved out of the Salt codebase but continue to be maintained by the Salt Core Team
 3. Files that will be moved out into a dump repository and will no longer be maintained by the Salt Core Team
+
+
+We'll use `git filter-repo` to migrate modules to a separate repositoy while
+retaining their git history. Once the migration has taken place, we'll do a
+`git rm` against the main salt repository.
+
+
+Install filter-repo (Debian)
+```
+sudo apt-get install -y git-filter-repo
+```
+
+Now make a fresh clone and run filter repo. Starting in the root of this repository.
+
+```
+FILESDIR=$(pwd)
+cd /tmp
+git clone git@github.com:saltstack/salt.git community-extensions
+cd community-extensions
+git filter-repo --dry --paths-from-file=$FILESDIR/community-ext-modules.txt
```

Comment:
community-ext-modules.txt was gotten from where ?

Certainly not present in $FILESDIR/community-ext-modules.txt without someone putting it there :)

Reply:
It says `Starting in the root of this repository.`

Reply:
Sorry missed it, might be more obvious if that was on a new line, my eye got the fresh clone part and didn't finish to the end.

That line has two different items going on in it, and it might be better to separate them by a fresh line